pkg/watermark/wmb/interface.go

```diff
@@ -0,0 +1,16 @@
+package wmb
+
+import "github.com/numaproj/numaflow/pkg/isb"
+
+// IdleManagement decides when to send a control message and also keeps track of idle watermark's offset.
+type IdleManagement interface {
+	// NeedToSendCtrlMsg validates whether to send a control message for the given partition or not.
```

Comment:
```suggestion

// IdleManagement decides when to send a control message and also keeps track of idle watermark's offset.
type IdleManager interface {
	// NeedToSendCtrlMsg validates whether to send a control message for the given partition or not.
```
as per @vigith comment, it should be IdleManager.